public/app/plugins/panel/stat/module.tsx

```diff
@@ -9,6 +9,21 @@ export const plugin = new PanelPlugin<StatPanelOptions>(StatPanel)
   .setPanelOptions(builder => {
     addStandardDataReduceOptions(builder);
 
+    builder.addSelect({
+      path: 'nameAndValue',
```

Comment:
I don't really like `nameAndValue` as a property -- maybe `showText` or `textMode`?

Reply:
changed to textMode